working/0110-incremental-sound-nnbd/roadmap.md

```diff
@@ -175,8 +175,15 @@ and faster code.
 
 Non-nullable types will be controlled by a language opt-in as
 described [elsewhere](https://github.com/dart-lang/language/issues/93).  In
-short, a library may opt in via syntax in the code, or a package may opt in in
-entirety.  Opting in applies only to the library or package so marked.
+short, a package may be opted-in and specific libraries within that package
+may be opted-out.
+
+Within a package that has been opted-in, specific libraries may be opted-out
+by adding `@pragma('nnbd-legacy')` to that library's `library` directive.
```

Comment:
Interesting choice. I'm not particularly happy about using pragmas that have semantic effects, but it would allow us to get the feature without adding any new language features for version-opt-in.

(We should allow library declarations with no name, so you can add annotations and comments on the library without having to give it a unique name).

Reply:
Per comments in https://github.com/dart-lang/language/issues/94, perhaps there is a better choice than @pragma. I'm happy with whatever mechanism is decided, but think it's important to have the ability to opt-out a library.